Source/WebCore/testing/Internals.mm

```diff
@@ -92,6 +92,10 @@ - (NSAttributedString *)_attributedStringForRange:(NSRange)range
 
 #endif // ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
 
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/WebCoreInternalsAdditions.mm>)
```

Comment:
I think it's generally okay to assume that a WebKitAdditions header will always be available if it's an internal build. Even if this is a header that needs to be added to WKA as part of this change.

Reply:
Agreed, but it allows to land this PR without waiting for the corresponding WKA PR.